internal/jennies/golang/builder.go

```diff
@@ -69,6 +68,12 @@ func (jenny *Builder) generateBuilder(context languages.Context, builder ast.Bui
 		buildObjectSignature = jenny.typeFormatter.variantInterface(builder.For.Type.ImplementedVariant())
 	}
 
+	constructorName := fmt.Sprintf("New%s", formatObjectName(builder.For.SelfRef.ReferredType))
```

Comment:
Nit: using straightfwd `+` for non-formatting string op:

```suggestion
	constructorName := "New" + formatObjectName(builder.For.SelfRef.ReferredType)
```